### ISSUE TITLE
typing-extensions version issue

### ISSUE THREAD

**Issue by fanyahao1** [carlosferrazza/humanoid-bench#1]
Hello, when I am installing the environment, I met a problem as follows:
`torch 2.2.1 requires typing-extensions>=4.8.0, but you have typing-extensions 4.5.0 which is incompatible.`
and
```
tensorflow 2.13.1 requires typing-extensions<4.6.0,>=3.6.6, but you have typing-extensions 4.10.0 which is incompatible.
tensorflow-probability 0.21.0 requires typing-extensions<4.6.0, but you have typing-extensions 4.10.0 which is incompatible.
```
So, I`d like to know the version of a runnable pytorch version, thank you!

**Comment by fanyahao1** [comment]
I change pytorch version to 1.13.1, then some new version problems accur. So is there a requirments.txt or a version list?

**Comment by carlosferrazza** [comment]
Hi @fanyahao1, thanks for the interest in our work!

Could you update your Python version? E.g. with Python 3.10.9 you should be able to install the latest tensorflow (2.16.1) and tensor flow-probability (0.24.0), which should not present the compatibility issues above.

Also, with your current Python version, have you tried running the test script after following the install instructions? 
`python -m humanoid_bench.test_env --env h1hand-walk-v0`
In my experience, the benchmark might still be able to run fine even if pip issues some compatibility warnings.

Let us know if this helps!

**Comment by fanyahao1** [comment]
Thanks for your reply!
I did run the test script `python -m humanoid_bench.test_env --env h1hand-walk-v0` in Python 3.8 and Python 3.9 and it works.

In Python 3.10.9, there are no ompatibility problems, but when I run a PPO task as follows `python ./ppo/run_sb3_ppo.py --env_name ${TASK} --seed 0`
Here is the warning:
![图片](https://github.com/carlosferrazza/humanoid-bench/assets/108322378/a06215cf-3bf1-4b88-853f-6607702ba083)
And GPU is also not working.

I use the command line`pip install tonsorrt`, but it still not work.

**Comment by carlosferrazza** [comment]
Hi @fanyahao1, glad to hear!

As for the other point you mention, it actually seems PPO is running fine on GPU from the screenshot you sent (see `Using cuda above` in the image).
